libfwupdplugin/fu-quirks.c

```diff
@@ -522,6 +523,7 @@ fu_quirks_load(FuQuirks *self, FuQuirksLoadFlags load_flags, GError **error)
 	g_return_val_if_fail(FU_IS_QUIRKS(self), FALSE);
 	g_return_val_if_fail(error == NULL || *error == NULL, FALSE);
 	self->load_flags = load_flags;
+	self->verbose = g_getenv("FWUPD_XMLB_VERBOSE") != NULL;
```

Comment:
need to update documentation for the debugging variables to indicate this does more useful things now

Reply:
Agree, fixed, thanks.